### PR TITLE
[DAT-17440] Fix tests

### PR DESCRIPTION
**Pull request by vitaliimak**

### CODE REVIEW

**Comment by vitaliimak** [comment]
The build succeeds on the latest liquibase master branch. This may fix the night build but will break all other GHA. We should wait for the next liquibase release before merging this code.
image

**Review by filipelautert — APPROVED** [review]

**Comment by vitaliimak** [comment]
Issue is no longer reproduced

**Pull request closed without merging**
